cve_bin_tool/extractor.py

```diff
@@ -46,7 +46,15 @@ def __init__(self, logger=None):
             self.extract_file_rpm: {".rpm"},
             self.extract_file_deb: {".deb", ".ipk"},
             self.extract_file_cab: {".cab"},
-            self.extract_file_zip: {".exe", ".zip", ".jar", ".apk", ".msi"},
+            self.extract_file_zip: {
+                ".exe",
+                ".zip",
+                ".jar",
+                ".apk",
+                ".msi",
+                ".egg",
+                ".whl",
```

Comment:
Here we are just using unzip because docs of wheel said:
### Usage
wheel unpack <wheel_file>
### Description
Unpack the given wheel file.

This is the equivalent of unzip <wheel_file>, except that it also checks that the hashes and file sizes match with those in RECORD and exits with an error if it encounters a mismatch.

Do we need to check for hash and file size mismatch? 
If yes, then we have to use wheel package instead of unzip.